File: works/tmp/2588.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().split('\n');
// let input = fs.readFileSync('input.txt').toString().split('\n');

A = parseInt(input[0])
B = parseInt(input[1])

console.log(A * (B % 10))
console.log(A * Math.floor(B/10%10))
console.log(A * Math.floor(B/100))
console.log(A*B)

File: works/tmp/2588.dart
import 'dart:io';

void main()
{
    int A = int.parse(stdin.readLineSync()!);
    int B = int.parse(stdin.readLineSync()!);

    print(A * (B % 10));
    print(A * (B/10%10).floor());
    print(A * (B/100).floor());
    print(A*B);

}